content/applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/resupply_warehouses.rst

```diff
@@ -1,70 +1,174 @@
-===============================
-Resupply from another warehouse
-===============================
+=============================
+Inter-warehouse replenishment
+=============================
 
-A common use case for multiple warehouses is to have one central warehouse that resupplies multiple
-shops, and in this case, each shop is considered a local warehouse. When a shop wants to replenish
-a product, the product is ordered to the central warehouse. Odoo allows the user to easily set
-which warehouse(s) can resupply another warehouse.
+.. |MTO| replace:: :abbr:`MTO (Make to Order)`
+
+When a business has multiple retail shops, in many cases, they resupply their stock from a shared,
```

Comment:
Quick comment: 
Resupplying from another warehouse isn't just a use case for retail, no?
It could also fall under 3PL, wholesale, or internal material/component storage for things like manufacturing ops, professional or high-end service facilities (e.g., biomed, lab corps,), etc. I think?

Reply:
I reworded the intro to address your comment (: